package/MDAnalysis/core/universe.py

```diff
@@ -465,7 +465,7 @@ def transfer_to_memory(self, start=None, stop=None, step=None, verbose=None, qui
             # fall back to a slower approach
             except AttributeError:
                 pm = ProgressMeter(self.trajectory.n_frames,
-                                   interval=step, verbose=verbose)
+                                   interval=1, verbose=verbose)
```

Comment:
In #960, I introduce the "progressive" scheme that writes all the frames from 0 to 20, then all ten frames from 20 to 200, ect... I would see that as a good default. In the meantime, an interval of 1 does the job.